reference/5.1/Microsoft.PowerShell.Core/About/about_Automatic_Variables.md

```diff
@@ -270,8 +270,7 @@ invoked, such as the name of the script that called the current command.
 
 `$MyInvocation` is populated only for scripts, function, and script blocks. You
 can use the information in the **System.Management.Automation.InvocationInfo**
-object that `$MyInvocation` returns in the current script, such as the path and
-filename of the script (`$MyInvocation.MyCommand.Path`) or the name of a
```

Comment:
This should have been `$MyInvocation.PSCommandPath`

Reply:
There already is a mention of `$MyInvocation.PSCommandPath` in the next paragraph.